src/Microsoft.AspNetCore.Sockets/Internal/FramingChannel.cs

```diff
@@ -43,6 +43,15 @@ ValueTask<Message> IReadableChannel<Message>.ReadAsync(CancellationToken cancell
             }
         }
 
+        private void CancelRead()
+        {
+            // We need to fake cancellation support until we get a newer build of pipelines that has CancelPendingRead()
```

Comment:
Yes there's a newer API but @pranavkm wrote some dubious logic and we're not getting updated pipeline packages.